pkg/process/metrics.go

```diff
@@ -48,3 +58,17 @@ func initMetrics(ctx context.Context, r *monkit.Registry, instanceID string) (
 	go c.Run(ctx)
 	return nil
 }
+
+// InitMetrics: Initialize telemetry reporting, using the node ID corresponding
+// to the given certificate as the telemetry instance ID.
+func InitMetricsWithCertPath(ctx context.Context, r *monkit.Registry, certPath string) error {
+	var metricsID string
+	nodeID, err := identity.NodeIDFromCertPath(certPath)
+	if err != nil {
+		zap.S().Errorf("Could not read identity for telemetry setup: %v", err)
+		metricsID = ""
```

Comment:
should we give it another option? seems odd to just leave it empty.

Reply:
InitMetrics() will use the hw MAC address by default if you pass in an empty string, which is the way it worked before. Is it confusing to defer the work to InitMetrics() like this?

Reply:
to clarify: using the MAC address is the way it worked before, not the part about how passing in an empty string makes it do that.

Reply:
Slightly, maybe a comment to explain what's happening downstream, just to prevent someone from pulling their hair out trying to figure out where the MAC address is getting set